src/Json.Schema.ToDotNet/ComparerCodeGenerator.cs

```diff
@@ -0,0 +1,1579 @@
+        /// </returns>
+        internal string Generate(string className, PropertyInfoDictionary propertyInfoDictionary)
+        {
+            _className = className;
+            _propertyInfoDictionary = propertyInfoDictionary;
+            _classType = SyntaxFactory.ParseTypeName(_className);
+            _localVariableNameGenerator.Reset();
+
+            string comparerClassName = GetComparerClassName(_className);
+            var comparerInterface = GetBaseType(_className, IComparerInterfaceName);
+
+            ClassDeclarationSyntax classDeclaration =
+                SyntaxFactory.ClassDeclaration(comparerClassName)
+                    .AddModifiers(
+                        SyntaxFactory.Token(SyntaxKind.InternalKeyword),
+                        SyntaxFactory.Token(SyntaxKind.SealedKeyword))
+                    .AddBaseListTypes(comparerInterface)
+                    .AddMembers(
+                        GenerateInstanceProperty(),
+                        GenerateCompareMethod());
+
+            var usings = new HashSet<string>
+            {
+                "System",                       // For Object.
+                "System.Collections.Generic"    // For IComparer<T>
+            };
+
+            IEnumerable<string> namespaceNames = _propertyInfoDictionary
+                .Values
+                .Select(propertyInfo => propertyInfo.NamespaceName)
+                .Where(namespaceName => !string.IsNullOrWhiteSpace(namespaceName));
+
+            foreach (string namespaceName in namespaceNames)
+            {
+                usings.Add(namespaceName);
+            }
+
+            return classDeclaration.Format(
+                _copyrightNotice,
+                usings,
+                _namespaceName,
+                MakeSummaryComment());
+        }
+
+        internal static BaseTypeSyntax GetBaseType(string className, string baseTypeName)
+        {
+            return SyntaxFactory.SimpleBaseType(
+                SyntaxFactory.GenericName(
+                    SyntaxFactory.Identifier(baseTypeName),
+                    SyntaxFactory.TypeArgumentList(SyntaxFactory.SeparatedList(
+                        new TypeSyntax[] {
+                            SyntaxFactory.ParseTypeName(className)}))));
+        }
+
+        internal static BaseTypeSyntax GetComparerBaseType(string className)
+        {
+            return GetBaseType(className, IComparerInterfaceName);
+        }
+
+        private string MakeSummaryComment()
+        {
+            return string.Format(
+                CultureInfo.CurrentCulture,
+                Resources.ComparerSummary,
+                _className);
+        }
+
+        private MemberDeclarationSyntax GenerateInstanceProperty()
+        {
+            TypeSyntax comparerType = SyntaxFactory.ParseTypeName(GetComparerClassName(_className));
+
+            // public static readonly ComparerType Instance = new ComparerType();
```

Comment:
>// public static readonly ComparerType Instance = new ComparerType(); [](http://example.com/codeflow?start=12&length=69)

remove all not needed comments

Reply:
I added this kind of comments demonstrating what is the generated code by the following code, since the syntax is not very readable. Other code generation class has the same comments.

Updated all the comments with this purpose with multi-lines comment block (/* ... */) and remove the unnecessary comments